[2025-09-30T17:15:38.405+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-30T17:15:38.579+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag2.task_pwd scheduled__2025-09-30T17:13:00+00:00 [queued]>
[2025-09-30T17:15:38.624+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag2.task_pwd scheduled__2025-09-30T17:13:00+00:00 [queued]>
[2025-09-30T17:15:38.629+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-09-30T17:15:38.726+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): task_pwd> on 2025-09-30 17:13:00+00:00
[2025-09-30T17:15:38.820+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=58) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-09-30T17:15:38.855+0000] {standard_task_runner.py:63} INFO - Started process 62 to run task
[2025-09-30T17:15:38.825+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag2', 'task_pwd', 'scheduled__2025-09-30T17:13:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/dag2.py', '--cfg-path', '/tmp/tmpo0y5__cp']
[2025-09-30T17:15:38.911+0000] {standard_task_runner.py:91} INFO - Job 4: Subtask task_pwd
[2025-09-30T17:15:39.032+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-09-30T17:15:39.342+0000] {task_command.py:426} INFO - Running <TaskInstance: dag2.task_pwd scheduled__2025-09-30T17:13:00+00:00 [running]> on host 7cd44293b4c0
[2025-09-30T17:15:39.949+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag2' AIRFLOW_CTX_TASK_ID='task_pwd' AIRFLOW_CTX_EXECUTION_DATE='2025-09-30T17:13:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-30T17:13:00+00:00'
[2025-09-30T17:15:39.962+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-30T17:15:39.971+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-30T17:15:39.983+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'pwd']
[2025-09-30T17:15:40.063+0000] {subprocess.py:86} INFO - Output:
[2025-09-30T17:15:40.079+0000] {subprocess.py:93} INFO - /tmp/***tmp3qa9cufr
[2025-09-30T17:15:40.082+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-09-30T17:15:40.089+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-30T17:15:40.284+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag2, task_id=task_pwd, run_id=scheduled__2025-09-30T17:13:00+00:00, execution_date=20250930T171300, start_date=20250930T171538, end_date=20250930T171540
[2025-09-30T17:15:40.434+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-09-30T17:15:40.508+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-30T17:15:40.539+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
